assets/scss/_card_dark.scss

```diff
@@ -0,0 +1,26 @@
+@mixin card_dark {
+  .card-small {
+    background-color: $bg-color-dark;
+    border: 1px solid $bg-color;
+  }
+
+  .card { border: 1px solid $bg-color; }
+
+  .card-image {
+    filter: grayscale($filter-percentage);
```

Comment:
This is cool but we should use the original image and don't force gray scale.

Reply:
Ok, I followed [this post](https://web.dev/prefers-color-scheme/#re-colorize-and-darken-photographic-images), thought it was a good idea to set it.

Reply:
Interesting. Weshould talk more about this later an maybe apply to other images too.

Reply:
Ok, do I have to remove it and create an issue with this feature ?

Reply:
It is ok to keep. No problem.